cmd/mb-main.go

```diff
@@ -33,6 +33,11 @@ var mbFlags = []cli.Flag{
 		Value: "us-east-1",
 		Usage: "specify bucket region; defaults to 'us-east-1'",
 	},
+	cli.StringFlag{
+		Name:  "panfs-path",
+		Value: "",
```

Comment:
Let's not use  `""` everywhere because it can mean anything and without context it is not clear what it means in a particular piece of code. Let's define some a friendly-named constant for this.

Reply:
Maybe it makes sense to remove value at all as we already have another mechanism to set default path to bucket

Reply:
@akhmialevskiy  One more question here - do we need to even mention in help message something about default value? As this default value is specified on the server side.

Reply:
> Maybe it makes sense to remove value at all as we already have another mechanism to set default path to bucket

I think that both options can be left. In the `minio`, some parameters can be set in two ways by env and as cli parameter

Reply:
> @akhmialevskiy One more question here - do we need to even mention in help message something about default value? As this default value is specified on the server side.

I don't think so, you are right.